README.md

```diff
@@ -19,9 +19,43 @@ memory_saver.pause()
 memory_saver.resume()
 ```
 
+## Installation
+
+For the library to work properly, it needs to be preloaded using the `LD_PRELOAD` environment variable.
```

Comment:
nit: would be great to mention the SGLang approach, which indeed does not require any operations